src/test/java/net/datafaker/providers/base/AwsTest.java

```diff
@@ -51,4 +51,10 @@ void testRegion() {
         String region = faker.aws().region();
         assertThat(region).matches("^[a-z]{2}-(south|east|north|west|northeast|central|southeast)-\\d$");
     }
+
+    @Test
+    void testService() {
+        assertThat(faker.aws().service()).isNotEmpty();
+    }
```

Comment:
If it's simply going to check not empty then I'd suggest using this type of test: https://github.com/datafaker-net/datafaker/blob/3a57f20c6603adec778ecb85de5e8cc525a98df0/src/test/java/net/datafaker/providers/base/ZodiacTest.java#L8-L12

Reply:
Sorry for the delay. 

Done